src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -204,6 +204,28 @@ public void runWithOverriddenNamespace() throws Exception {
         r.assertLogContains(overriddenNamespace, b);
     }
 
+    @Test
+    public void runWithOverriddenNamespace2() throws Exception {
```

Comment:
should this just be the only `runWithOverriddenNamespace` test ?
I'm not sure what the other one was testing because the namespace was not in the pipeline